developer_manual/general/codingguidelines.rst

```diff
@@ -40,9 +40,12 @@ The most important labels and their meaning:
  * #6 - Reviewing - review in progress
  * #7 - To Release - reviewed PR that awaits unfreeze of a branch to get merged
 * App tags: #app:files #app:user_ldap #app:files_versions and so on. These tags indicate the app that is impacted by the issue or which the PR is related to
-* settings tags: #settings:personal #settings:apps #settings:admin and so on. These tags indicate the settings area that is impacted by the issue or which the PR is related to
+* Settings tags: #settings:personal #settings:apps #settings:admin and so on. These tags indicate the settings area that is impacted by the issue or which the PR is related to
 * db tags: #db:mysql #db:sqlite #db:postgresql and so on. These tags indicate the database that is impacted by the issue or which the PR is related to
-* browser tags: #browser:ie #browser:safari and so on. These tags indicate the browser that is impacted by the issue or which the PR is related to
+* Browser tags: #browser:ie #browser:safari and so on. These tags indicate the browser that is impacted by the issue or which the PR is related to
+* Component tags: #comp:filesystem #comp:javascript and so on. These tags indicate the components of ownCloud impacted by the issue or which the PR is related to
+* Development tool tags: #dev:unit_testing #dev:public_API and so on. These tags indicate the development tool (and not ownCloud code) impacted by the issue or which the PR is related
```

Comment:
This is owncloud (app) development related - external apps (like our mobile app that uses the public OCS API), owncloud apps (like any app that uses stuff like OCP\... or AppFramework) or internal stuff (like unit tests or JS tests)